bin/fetch-blocklist

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+set -eu
+
+if [ -z "${VIA_BLOCKLIST_URL:-}" ]; then
+  echo "Skipping blocklist fetch. No VIA_BLOCKLIST_URL env var is configured."
+  exit 0
+fi
+
+if [ -z "${VIA_BLOCKLIST_PATH:-}" ]; then
+  echo "Skipping blocklist fetch. No VIA_BLOCKLIST_PATH env var is configured."
+  exit 0
+fi
+
+echo "Fetching blocklist from $VIA_BLOCKLIST_URL as $VIA_BLOCKLIST_PATH"
+curl --silent --max-time 5 $VIA_BLOCKLIST_URL > $VIA_BLOCKLIST_PATH.new
+mv $VIA_BLOCKLIST_PATH.new $VIA_BLOCKLIST_PATH
```

Comment:
:+1: Seems simple enough. I guess this blocklist'll be getting downloaded by every app server